poi-4.1.2/pom.xml

```diff
@@ -201,6 +203,18 @@
             <version>1.05</version>
             <optional>false</optional>
         </dependency>
+        <dependency>
+            <groupId>org.apache.xmlgraphics</groupId>
```

Comment:
No need to add the dependency if it's already a transitive dependency from POI.

Reply:
It's not actually listed as a dependency in the poi-ooxml pom for some reason, so isn't pulled in transitively. It is a compile dependency, though: https://github.com/apache/poi/blob/REL_4_1_2/build.gradle#L243

I noticed the same is true for the bouncy castle dependency (used at compile-time, missing in the POI pom), and it's not included here as a dependency, so went ahead and removed batik-all anyway